### PR TITLE
Do not register the file whith asset bundle if is null

### PR DESCRIPTION
**Pull request by Razzwan**
| Q | A |
| --- | --- |
| Is bugfix? | yes |
| New feature? | no |
| Breaks BC? | no |
| Tests pass? | no |
| Fixed issues | empty |

I whant to register the asset file in my develop environment only and I'm writing:

``` php
<?php
namespace common\assets;

use yii\web\AssetBundle;

/**
 * @author Razzwan Lomov <razvanlomov@gmail.com>
 * @since  2.1
 */
class ReactAsset extends AssetBundle
{
    public $sourcePath = null;

    public $js = [
        YII_ENV_DEV ? "//fb.me/react-15.0.1.js" : "//fb.me/react-15.0.1.min.js",
        YII_ENV_DEV ? "//fb.me/react-dom-15.0.1.js" : "//fb.me/react-dom-15.0.1.min.js",
        YII_ENV_DEV ? "//cdnjs.cloudflare.com/ajax/libs/babel-core/5.6.15/browser.js" : null,
    ];
}
```

But it work wrong. I propose to fix it.

### CODE REVIEW

File: framework/web/AssetBundle.php [hunk 1]
```diff
@@ -153,7 +153,9 @@ public function registerAssetFiles($view)
                 $options = ArrayHelper::merge($this->jsOptions, $js);
                 $view->registerJsFile($manager->getAssetUrl($this, $file), $options);
             } else {
-                $view->registerJsFile($manager->getAssetUrl($this, $js), $this->jsOptions);
+                if (!is_null($js)) {
```
Comment (samdark) [inline framework/web/AssetBundle.php]: `$js !== null`

File: framework/web/AssetBundle.php [hunk 2]
```diff
@@ -162,7 +164,9 @@ public function registerAssetFiles($view)
                 $options = ArrayHelper::merge($this->cssOptions, $css);
                 $view->registerCssFile($manager->getAssetUrl($this, $file), $options);
             } else {
-                $view->registerCssFile($manager->getAssetUrl($this, $css), $this->cssOptions);
+                if (!is_null($css)) {
```
Comment (samdark) [inline framework/web/AssetBundle.php]: `$js !== null`

**Comment by lynicidn** [comment]
1. https://github.com/yiisoft/yii2/blob/master/docs/guide/structure-assets.md#customizing-asset-bundles-

2. 
```php
public functuion init() 
{
    if (Yii_DEBUG) {
        $this->js[] = 
    }
}
```

3. not ice 
```php
public functuion register($view) 
{
    parent::register($view);
    if (Yii_DEBUG) {
        //$view->registerJsFile()
    }
}
```

**Comment by Razzwan** [comment]
What else should I do to this request took? This is my first attempt to participate in the joint development, perhaps I do not understand something.

**Comment by samdark** [comment]
Everything's fine. Merged. Thank you.

**Pull request closed without merging**